build.sbt

```diff
@@ -219,4 +225,4 @@ checkCodeFormat := {
   }
 }
 
-addCommandAlias("validateCode", ";scalariformFormat;checkCodeFormat")
+addCommandAlias("validateCode", ";scalariformFormat;headerCreate;checkCodeFormat")
```

Comment:
Need to add `test:scalariformFormat` and `test:headerCreate` (`headerCheck;test:headerCheck`).

Reply:
thanks